transports/webrtc-utils/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+edition = "2021"
+license = "MIT"
+name = "libp2p-webrtc-utils"
+rust-version.workspace = true
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bytes = "1"
+futures = "0.3"
+hex = "0.4"
+js-sys = { version = "0.3", optional = true }
+libp2p-core = { workspace = true }
+libp2p-identity = { workspace = true }
+libp2p-noise = { workspace = true }
+log = "0.4.19"
+quick-protobuf = "0.8"
+quick-protobuf-codec = { workspace = true }
+serde = { version = "1.0", features = ["derive"] }
+sha2 = "0.10.7"
+thiserror = "1"
+tinytemplate = "1.2"
+wasm-bindgen = { version = "0.2.87", optional = true }
+web-sys = { version = "0.3.64", optional = true, features = [
+    "RtcSdpType",
+    "RtcSessionDescription",
+    "RtcSessionDescriptionInit",
+] }
```

Comment:
Please use single line to be consistent.

Reply:
Can do. But what happens when you have several (more than 3) features and the line becomes really long? (like here: https://github.com/DougAnderson444/rust-libp2p/blob/66bbf6c5638fa9b66f35ffb6e9fdfe64fdc37bd8/transports/webrtc-websys/Cargo.toml#L45)

I have been using `Even better TOML` for formatting and it usually puts long lines on multiple lines. I can override this manually I think

Reply:
> I have been using `Even better TOML` for formatting and it usually puts long lines on multiple lines. I can override this manually I think

Happy to have and enforce (!) a uniform toml format. But let's do that as part of a separate pull request. Help on https://github.com/libp2p/rust-libp2p/issues/4043 is very much appreciated.